build.gradle

```diff
@@ -30,6 +30,9 @@ if (System.env.CIRCLE_TEST_REPORTS) {
 group 'com.palantir.gradle.docker'
 version System.env.CIRCLE_TAG ?: gitVersion()
 
+sourceCompatibility = 1.7
+targetCompatibility = 1.7
```

Comment:
ok, let's do sourceC only, please. (since we usually only specify that in our projects.)